packages/arb-validator/cmdhelper/cmdhelper.go

```diff
@@ -116,7 +118,12 @@ func ValidateRollupChain(execName string, managerCreationFunc func(rollupAddress
 
 	validatorFolder := validateCmd.Arg(0)
 	ethURL := validateCmd.Arg(1)
-	addressString := validateCmd.Arg(2)
+	var addressString string
+	if addr == "" {
+		addressString = validateCmd.Arg(2)
+	} else {
+		addressString = addr
+	}
```

Comment:
Rather than doing this, instead have an inner `ValidateRollupChain` that takes all its parameter from arguments without the command line parsing then have a couple wrapper functions around that.